corehq/apps/es/es_query.py

```diff
@@ -533,12 +535,11 @@ class ESQuerySet(object):
 
     def __init__(self, raw, query):
         if 'error' in raw:
-            msg = ("ElasticSearch Error\n{error}\nIndex: {index}"
-                   "\nQuery: {query}").format(
-                       error=raw['error'],
-                       index=query.index,
-                       query=query.dumps(pretty=True),
-                    )
+            msg = textwrap.dedent(f"""ElasticSearch Error
+                {raw['error']}
+                Index: {query.index}
+                Query: {query.dumps(pretty=True)}""")
```

Comment:
Dedent will not do anything if any of the lines (like this first line) has no leading whitespace.
```suggestion
            msg = textwrap.dedent(f"""
                ElasticSearch Error
                {raw['error']}
                Index: {query.index}
                Query: {query.dumps(pretty=True)}""").lstrip()
```

Reply:
Ahh! I see. Thanks for the suggestion.